web/beacon-app/src/features/auth/routes/LoginPage.tsx

```diff
@@ -51,20 +62,53 @@ export function Login() {
   }, [param?.accountVerified]);
 
   useEffect(() => {
-    if (!isAuthenticated(login)) {
+    if (!isAuthenticated(authenticate)) {
       clearSessionStorage();
     }
-  }, [login]);
+  }, [authenticate]);
 
   useEffect(() => {
-    if (login.authenticated) {
-      const token = decodeToken(login?.auth?.access_token) as any;
-      Store.setAuthUser(token, !!login.authenticated);
+    if (authenticated) {
+      setCurrentUserEmail('');
+      const token = decodeToken(auth?.access_token) as any;
+      Store.setAuthUser(token, !!authenticated);
       removeCookie('invitee_token');
       navigate(APP_ROUTE.DASHBOARD);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [login.authenticated, navigate, login?.auth?.access_token]);
+  }, [authenticated, navigate, auth?.access_token]);
+
+  useEffect(() => {
+    if (error && error.response.status === 403) {
+      // if error is 400 display a toast with a resend email button
```

Comment:
The check on line 82 is for a `403`, but the comment on line 83 states that the toast should display if there's a 400. Should the comment be updated?